Examples/Covasim/environment.py

```diff
@@ -47,26 +46,35 @@ def __init__(self, data = {
 
         number_of_windows = int((datetime.datetime.strptime(end_date, "%Y-%m-%d") - datetime.datetime.strptime(start_date, "%Y-%m-%d")).days/self.window)
         self.duration = self.window * number_of_windows
-        self.action_space = gym.spaces.MultiDiscrete([20]*number_of_windows+[20])
+        self.action_names = np.array(["beta"]*number_of_windows+["d0"])
+        self.actions_start_dates = [(datetime.datetime.strptime(start_date, "%Y-%m-%d") + datetime.timedelta(i * self.window)).strftime("%Y-%m-%d") for i in range(number_of_windows)]+[start_date]
+        self.actions_end_dates = [(datetime.datetime.strptime(start_date, "%Y-%m-%d") + datetime.timedelta(((i+1) * self.window)-1)).strftime("%Y-%m-%d") for i in range(number_of_windows)] \
+        +[(datetime.datetime.strptime(start_date, "%Y-%m-%d") + datetime.timedelta(self.duration-1)).strftime("%Y-%m-%d")]
+
+        self.action_space = gym.spaces.Box(
+            low=np.array([0.0001]*number_of_windows+[0.5]), 
+            high=np.array([0.01]*number_of_windows+[1]), dtype=float)
```

Comment:
Even though it's an early example, suggesting:
`high=np.array([0.05]*number_of_windows+[10]), dtype=float)`